community-membership.md

```diff
@@ -97,16 +97,16 @@ approver in addition to the reviews by *members.*
 
 ## Triager
 
-Triagers assist the maintainers and approvers with project management and 
-backlog organization. The specific workflows and triage requirements depend on 
+Triagers assist the maintainers and approvers with project management and
+backlog organization. The specific workflows and triage requirements depend on
 the project, and are set by the project maintainers.
 
-Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level), 
-with the names of the current Triagers commited to git, either in CONTRIBUTING, 
+Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level),
+with the names of the current Triagers commited to git, either in CONTRIBUTING,
 CODEOWNERS, or the botom of the README.
```

Comment:
Not introduced by this PR: s/botom/bottom/

Reply:
```suggestion
CODEOWNERS, or the bottom of the README.
```